src/au3cloud/internal/au3audiocomservice.cpp

```
@@ -128,17 +147,94 @@ au::au3cloud::Err uploadResultToErr(UploadOperationCompleted::Result result)
+}
+
+void Au3AudioComService::init()
+{
+    globalContext()->currentProjectChanged().onNotify(this, [this]() {
+        auto project = globalContext()->currentProject();
+        if (!project) {
+            return;
+        }
+
+        project->aboutCloseBegin().onNotify(this, [w_project = std::weak_ptr(project)]() {
+            auto project = w_project.lock();
```

Cppcheck (2.20.0)

[style] The function 'toProjectFile' is never used.

(unusedFunction)